src/java.base/share/classes/jdk/internal/util/OctalDigits.java

```diff
@@ -34,14 +34,20 @@
  *
  * @since 21
  */
-final class OctalDigits implements Digits {
-    @Stable
-    private static final short[] DIGITS;
-
+public final class OctalDigits implements Digits {
```

Comment:
Can you restore the order of methods here to minimize the number of lines of code are touched?